tuxemon/map.py

```diff
@@ -447,12 +448,28 @@ def __init__(
         self.maps = maps
 
         # optional fields
+        self.slug = maps.get("slug")
+        self.name = T.translate(self.slug)
+        self.description = T.translate(f"{self.slug}_description")
+        # cardinal directions (towns + roads)
+        self.north = maps.get("north")
+        self.south = maps.get("south")
+        self.east = maps.get("east")
+        self.west = maps.get("west")
+        # translated cardinal directions (signs)
+        self.cnorth = T.translate(self.north)
```

Comment:
The c prefix is a bit cryptic. Maybe north_translated? north_trans?

Reply:
Yeah, fair point, I'll update